docs/source/tutorials/Using MultiArmedBandits.md

```diff
@@ -0,0 +1,19 @@
+### Using UCB agent on a Bernoulli Multi Armed Bandit
+
+```python
+import gym
+
+from genrl.bandit import UCBMABAgent, BernoulliMAB, MABTrainer
+
+bandit = BernoulliMAB(bandits=10, arms=5, context_type="int")
+agent = UCBMABAgent(bandit, confidence=1.0)
+
+trainer = MABTrainer(agent, bandit)
+trainer.train(timesteps=10000)
+```
+
+Implementation of the bandit:
+https://github.com/SforAiDl/genrl/blob/master/genrl/bandit/bandits/multi_armed_bandits/bernoulli_mab.py
+
+Implementation of the agent:
+https://github.com/SforAiDl/genrl/blob/master/genrl/bandit/agents/mab_agents/ucb.py
```

Comment:
Can you include a link to the docs instead of this. You an look at the Using A2C docs as a example.

Reply:
The links in the Using A2C doc are giving an Error 404.
What should I put the link to?

Reply:
Woring fine for me. https://genrl.readthedocs.io/en/latest/usage/tutorials/Using%20A2C.html

Reply:
Try in incognito. It might be loading the old one?

Reply:
The link in the doc in this repo is not working [https://github.com/SforAiDl/genrl/blob/master/docs/source/api/algorithms/genrl.deep.agents.a2c.html](https://github.com/SforAiDl/genrl/blob/master/docs/source/api/algorithms/genrl.deep.agents.a2c.html)

Reply:
Yeah, thats happening because its a relative link. In this, you can just link it to  the docs directly. E.g. for bernoulli MAB you can link https://genrl.readthedocs.io/en/latest/api/bandit/genrl.bandit.bandits.multi_armed_bandits.html#genrl.bandit.bandits.multi_armed_bandits.bernoulli_mab.BernoulliMAB

Reply:
I've done this in the last commit.